test/properties.ts

```diff
@@ -119,4 +122,52 @@ test('configured options', async (t) => {
     });
 
     return lock.latch;
+});
+
+
+
+test('noEcho', async(t) => {
+    t.plan(1);
+    let lock = new Lock();
+    let sc = t.context as SC;
+    let subj = createInbox();
+    let cp = connect({url: sc.server.nats, noEcho: true});
+    cp.then(async (nc) => {
+        let c2 = 0;
+        let sub2 = nc.subscribe(subj, ()=> {
+            c2++;
+        });
+        nc.publish(subj);
+        await nc.flush();
+        t.is(c2, 0);
+        nc.close();
+        lock.unlock();
+    }).catch((err) => {
+        if(err.code === ErrorCode.NO_ECHO_NOT_SUPPORTED) {
```

Comment:
I assume this is in case the tests runs against an older server, right?

Reply:
yeap